tests/conftest.py

```diff
@@ -231,13 +230,13 @@ def verify_url(request: FixtureRequest, base_url: str):
         session.get(base_url, verify=False)
 
 
-@pytest.fixture
+@pytest.fixture()
```

Comment:
What motivated these changes?

Reply:
# pytest-fixture-incorrect-parentheses-style (PT001)

Derived from the **flake8-pytest-style** linter.

Fix is always available.

## What it does
Checks for argument-free `@pytest.fixture()` decorators with or without
parentheses, depending on the `flake8-pytest-style.fixture-parentheses`
setting.

## Why is this bad?
If a `@pytest.fixture()` doesn't take any arguments, the parentheses are
optional.

Either removing those unnecessary parentheses _or_ requiring them for all
fixtures is fine, but it's best to be consistent.

## Example
```python
import pytest


@pytest.fixture
def my_fixture():
    ...
```

Use instead:
```python
import pytest


@pytest.fixture()
def my_fixture():
    ...
```

## Options
- `flake8-pytest-style.fixture-parentheses`

## References
- [`pytest` documentation: API Reference: Fixtures](https://docs.pytest.org/en/latest/reference/reference.html#fixtures-api)